core/domain/services/assets/CoreAssetManager.php

```diff
@@ -475,17 +386,23 @@ private function loadCoreCss()
             CoreAssetManager::CSS_HANDLE_CORE_CSS_DEFAULT,
             $this->registry->getCssUrl(
                 $this->domain->assetNamespace(),
-                'core-default-theme'
+                CoreAssetManager::CSS_HANDLE_CORE_CSS_DEFAULT
             ),
             ['dashicons']
         );
         $this->addStylesheet(
             CoreAssetManager::CSS_HANDLE_COMPONENTS,
```

Comment:
could this not use the new `addCss()` method you added?

Reply:
Yup, see 3cc025c